lib/rules/no-computed-properties-in-native-classes.js

```diff
@@ -53,7 +53,11 @@ module.exports = {
   },
 
   create(context) {
-    const ignoreClassic = context.options[0] && context.options[0].ignoreClassic;
+    const { options = [] } = context || {};
+    const [namedOptions = {}] = options;
+    const ignoreClassic = Reflect.has(namedOptions, 'ignoreClassic')
+      ? namedOptions.ignoreClassic
+      : true;
```

Comment:
We could replace all five of these lines with just this:

```js
const ignoreClassic = !context.options[0] || context.options[0].ignoreClassic;
````

(this is the same as what we do with the `allowDynamicKeys` option in the `require-computed-property-dependencies` rule which also defaults to true)

Reply:
yes I like this much better. I was sitting here thinking "I should be able to write this simpler, but it's just not coming"